### DISCUSSION TITLE
pimcore_portal_engine.elasticsearch.host

### DISCUSSION THREAD

**Discussion by esorthapa** [pimcore/pimcore#10447] [Q&A / Support]
Hi, 
 I am having problem during install pimcore portal engine in enterprise pimcore 6.9. Also I had installed l Elasticsearch  in my server too.
Error:
[ERROR] Required parameter "pimcore_portal_engine.elasticsearch.host" is not set. Overwrite this parameter in your yml 
         config. Read docs for more information

ANY HELP would be appreciated - even a link that spells it out would be great.

**Comment by fashxp** [comment]
Hi,
see https://pimcore.com/docs/portal-engine/current/Installation/Installation.html#page_Required-Parameter

**Reply by BjBue** [reply]
This link is a 404.
Would be great if you would fix it!
Have the same problem

**Reply by BjBue** [reply]
It would be more easy if you explain that the config on this page:
https://pimcore.com/docs/platform/Portal_Engine/Installation/
in the 
Setup Elasticsearch
should be made in 
/config/config.yaml
and yes, it should be clear, but why don`t write it in the documentation?

**Comment by esorthapa** [comment]
Sorry for the late reply.

I will send sample configuration regarding my.

Kind regards
Ishwar
